frontend/src/components/common/form/StepForm/__snapshots__/SteppedForm.test.tsx.snap

```diff
@@ -32,7 +32,7 @@ exports[`SteppedForm component renders correctly 1`] = `
 }
 
 .c0 .nav-tabs .nav-item.active svg {
-  background-color: #428bca;
+  background-color: secondaryVariantColor;
```

Comment:
interestingly the variables names appear in the snapshots now

Reply:
that actually is nice, since I'm not such a web developer that hash codes are meaningful to me.